pkg/package-server/apis/operators/packagemanifest.go

```diff
@@ -49,6 +51,22 @@ func CreateCSVDescription(csv *operatorsv1alpha1.ClusterServiceVersion, csvJSON
 		desc.Icon = icons
 	}
 
+	desc.Links = make([]AppLink, len(csv.Spec.Links))
```

Comment:
Can we not use [copy(dst, src []Type)](https://golang.org/pkg/builtin/#copy) here? Seems like both the slices are of the same type.

Reply:
They're similar but one of the types is provided via the packagemanfiest_types.go and the other is provide via `clusterserviceversion_types.go`.

Reply:
Can we abstract this out to another function at least then? Seems like we're writing the same code in 4 places.

Reply:
The same code is written in two places on purpose. OLM stores different versions of the `PackageManifest` type (internal/v1alpha1).